docs/Makefile

```diff
@@ -5,8 +5,8 @@
 # from the environment for the first two.
 SPHINXOPTS    ?=
 SPHINXBUILD   ?= sphinx-build
-SOURCEDIR     = .
-BUILDDIR      = _build
+SOURCEDIR     = source
```

Comment:
I changed the entry directories.  `source` is most standard in many OSS.

Reply:
Good cleanup!